commands.js

```diff
@@ -2476,8 +2476,8 @@ exports.commands = {
 		if (!this.can('hotpatch')) return false;
 		if (Monitor.hotpatchLock) return this.errorReply("Hotpatch is currently been disabled. (" + Monitor.hotpatchLock + ")");
 
-		let staff = Rooms('staff');
-		if (staff) staff.add("(" + user.name + " used /hotpatch " + target + ")").update();
+		let announceRoom = Rooms('development') ? Rooms('development') : Rooms('staff');
```

Comment:
This is exactly the situation `||` is for:

    let announceRoom = Rooms('development') || Rooms('staff');